tests/integration/suite/test_node.py

```diff
@@ -160,13 +160,22 @@ def _node_template():
     node_template = wait_for(_node_template,
                              fail_handler=lambda:
                              'failed to create node template')
-    wait_remove_resource(node_template)
 
     node_pool = client.create_node_pool(
         nodeTemplateId=node_template.id,
         hostnamePrefix="test1",
         clusterId="local")
+
+    def node_available():
+        node = client.list_node(nodePoolId=node_pool.id)
+        if len(node.data):
+            return node.data[0]
+        return None
```

Comment:
nit: Python returns `None` by default so this isn't necessary (unless you added it for readability?)
```suggestion
```